clinica/pipelines/t1_freesurfer_atlas/t1_freesurfer_atlas_utils.py

```diff
@@ -1,16 +1,96 @@
 # coding: utf8
 
 
+from pathlib import Path
+
 from clinica.utils.stream import cprint
 
 
+def path_cross(caps_directory: str, to_process_with_atlases: tuple, path: Path):
+    import os
+
+    hemisphere = Path(path.stem).stem
+    atlas_name = Path(path.stem).suffix[1:].split("_")[0]
```

Comment:
I don't really understand this line. [`stem`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem) is supposed to return the name of the file without the extension and [`suffix`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) the extension? 

The processing seems overly specific for a particular type of path/file name. Do you confirm it is generalizable?

Reply:
the freesurfer atlases have the format `hemisphere.atlas_name.extension` so it is generalizable ! 
If someone generates a freesurfer atlas, the format shouldn't be different, since it comes from Freesurfer. But we should write it in the guide though, just to make sure.

Reply:
I just checked, and the format of the atlas was already written in the documentation !